terraform/modules/mqtt/main.tf

```diff
@@ -0,0 +1,39 @@
+resource "terraform_data" "instance" {
+  # Store zone since variables not accessible at destroy time.
+  input = var.zone
+  provisioner "local-exec" {
+    command = join(" ", [
+      "gcloud compute instances create-with-container mqtt",
+      "--container-env",
+      join(",", [
+        "MQTT_PASSWORD=${var.mosquitto_password}",
+        "DATABASE_URL=${var.db_conn_str_private}"
+      ]),
+      "--container-image econialabs/inbox:mqtt",
+      "--network ${var.sql_network_id}",
+      "--zone ${var.zone}"
+    ])
+  }
+  provisioner "local-exec" {
+    command = join("\n", [
+      "result=$(gcloud compute instances list --filter NAME=mqtt)",
+      "if [ -n \"$result\" ]; then",
+      join(" ", [
+        "gcloud compute instances delete mqtt",
+        "--quiet",
+        "--zone ${self.output}"
+      ]),
+      "fi"
+    ])
+    when = destroy
+  }
+}
+
+data "external" "ip" {
```

Comment:
Is it possible to make this a property of the resource, perhaps specified at creation time?

Reply:
What does that mean ? This is just copied over from the Econia V4 repo.

Reply:
I was wondering if maybe this could be included in the instance that starts the container

But if it wasn't done in the Econia v4 repo, then maybe not, and we already know it works